fossunited/chapters/web_template/foss_clubs_home_page/foss_clubs_home_page.html

```diff
@@ -0,0 +1,409 @@
+<style>
+
+    .club-hero-section{
+        display: flex;
+        font-family: Inter;
+        overflow: hidden;
+        padding: 0rem;
+    }
+
+    .hero-container{
+        display: flex;
+        flex-direction: column;
+        gap: 3rem;
+
+    }
+
+    .hero-header{
+        display: grid;
+        grid-template-columns: 1fr;
+        gap: 1rem;
+
+    }
+
+    .hero-header-heading{
+        display: flex;
+        gap: 1rem;
+    }
+
+    .hero-header img{
+        height: 4rem;
+        width: auto;
+    }
+
+    h1{
+        margin:0;
+        color: #1A1A1A;
+        font-size: 4rem;
+        font-weight: 700;
+        line-height: 100%;
+        letter-spacing: -0.18rem;
+    }
+
+    h1 > span{
+        color: #08B74F;;
+    }
+
+    h5{
+    color: var(--Foundation-Code-Night-code-night-500, #1A1A1A);
+    font-size: 1.15rem;
+    font-weight: 600;
+    line-height: 115%;
+    letter-spacing: -0.06rem;
+    }
+
+    p{
+        color: var(--Gray-500, #667085);
+        font-size: 1rem;
+        font-style: normal;
+        font-weight: 500;
+        line-height: 140%;
+        letter-spacing: -0.04rem;
+    }
```

Comment:
Rather than overriding p and h tags, try using divs with classnames. I feel that would be cleaner and safer

Reply:
Yes, that's better....I have actually taken that way in the latter PRs. This one - I'll refactor after.